ui-admin/src/portal/MailingListView.tsx

```diff
@@ -108,26 +110,32 @@ export default function MailingListView({ portalContext, portalEnv }:
     setShowDeleteConfirm(false)
   }
 
-  return <div className="container p-3">
-    <h1 className="h4">Mailing list </h1>
+  return <div className="container-fluid px-4 py-2">
+    <div className="d-flex mb-2">
+      <h2 className="fw-bold">Mailing List</h2>
+    </div>
     <LoadingSpinner isLoading={isLoading}>
-      <div className="d-flex align-items-center">
-        <div>
-          {numSelected} of {table.getPreFilteredRowModel().rows.length} selected
+      <div className="d-flex align-items-center justify-content-between">
+        <div className="d-flex">
+          <RowVisibilityCount table={table}/>
+        </div>
+        <div className="d-flex">
+          <Button onClick={download}
+            variant="light" className="border m-1" disabled={!numSelected}
+            tooltip={numSelected ? 'Download selected contacts' : 'You must select contacts to download'}>
```

Comment:
Nit, and not new for this PR, but unclear why we are calling the entries in the mailing list 'contacts'. Do we use that term elsewhere?

Reply:
I think this is the only place where we use that term in the UI, b/c this is the only place in the UI where we show the mailing list.   The entities that back this list are called `MailingListContact`--these are different than "participants" -- these are just people who entered their email on the website because they wanted to get on the mailing list.  So 'contacts' seemed like the best term.